R/html-tweak.R

```diff
@@ -71,6 +71,32 @@ tweak_md_links <- function(html) {
   invisible()
 }
 
+tweak_all_links <- function(html, pkg = pkg) {
+  links <- xml2::xml_find_all(html, ".//a")
+  if (length(links) == 0)
+    return()
+
+  hrefs <- xml2::xml_attr(links, "href")
+
+  if (is.null(pkg$meta$url)) {
```

Comment:
Do we actually use this to create absolute urls?

Reply:
No but in e.g. the README there are absolute URLs (to the homepage, to the code of conduct) https://github.com/r-lib/pkgdown#pkgdown- so one needs to know the URL to not detect those as external links

Reply:
Ah good point. A comment mentioning that would be useful for future-me 😄

Reply:
The comment currently tells me what the code does — I want to know _why_ we need to do it (i.e. READMEs often include absolute urls for code of conduct etc)